lib/environment.rb

```diff
@@ -199,6 +199,17 @@ def idp
       return @idp
     end
 
+    def ocm_env
+      unless @ocm_env
+        if opts[:ocm_env]
+          @ocm_env = opts[:ocm_env]
+        else
+          @ocm_env = ''
+        end
+      end
+      return @ocm_env
+    end
```

Comment:
@pruan-rht , @xueli181114 , what is this method doing? where is it used? And for what? It can be shortened to 
```
def ocm_env
  @ocm_env = "#{opts[:ocm_env]}"
end
```
But I'd like to review how we use it.

Reply:
@akostadinov This method is used before, it is useless now, I will clean them up in next PR.